yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ErrorDetails.java

```diff
@@ -0,0 +1,94 @@
+package com.yoti.api.client.spi.remote.call;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class ErrorDetails {
+
+    @JsonProperty(Property.ERROR_CODE)
+    private String code;
+
+    @JsonProperty(Property.DESCRIPTION)
+    private String description;
+
+    private ErrorDetails() { }
+
+    private ErrorDetails(Builder builder) {
+        this.code = builder.code;
+        this.description = builder.description;
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public String getCode() {
+        return code;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        ErrorDetails that = (ErrorDetails) o;
+        return Objects.equals(code, that.code) && Objects.equals(description, that.description);
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+
+        result = prime * result + ((code == null) ? 0 : code.hashCode());
+        result = prime * result + ((description == null) ? 0 : description.hashCode());
+
+        return result;
+    }
+
+    @Override
+    public String toString() {
+        return String.format("- Error[code='%s', description='%s']", code, description);
+    }
+
+    public static final class Builder {
```

Comment:
Shouldn't this builder be used to de-serialize a JSON object? The class isn't specifying it with `@JsonDeserialize`